src/roboteam_ai.cpp

```diff
@@ -75,7 +75,7 @@ int main(int argc, char* argv[]) {
     rtt::SETTINGS.setVisionIp("127.0.0.1");
     rtt::SETTINGS.setVisionPort(10006);
     rtt::SETTINGS.setRefereeIp("224.5.23.1");
-    rtt::SETTINGS.setRefereePort(10007);
+    rtt::SETTINGS.setRefereePort(10003);
```

Comment:
Don't know if you should change this or not. Mine works at 10007?

Reply:
mine at 10003 as well